security/web-security.md

```diff
@@ -67,14 +77,16 @@ Follow [this Khan Academy course, Cybersecurity 101](https://www.khanacademy.org
 6. The next step is to try to XSS each other's sites.
```

Comment:
Could you help us by elaborating on steps 5 and 6?  If someone had never heard about security before, they wouldn't know what to do here.